script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -53,16 +53,16 @@ Wait a minute, what are you making me say, Tohno-kun!?"
 <Page15>
 "No, I wasn't trying to... Senpai, are you perhaps not a morning person?"
 "Yes. Actually, I'm pretty bad at waking up early.
-When I'm volunteering, I tend to be pretty reckless."
+When I'm volunteering, I tend to barely scrape by."
 <Page16>
-"My parents owned a bakery, so I used to wake up very early every day. Unfortunately, no matter how I tried I always struggled in the morning. My father has been driven mad by it ever since I was a kid."
+"My parents owned a bakery, so I used to wake up very early every day. Unfortunately, no matter how I tried, I always struggled in the morning. My father has been driven mad by it ever since I was a kid."
 <Page17>
 What? A bakery!?
 Arihiko and I look at each other. I bet he's also imagining Ciel-senpai with an oven in her room, kneading and baking flour with her white fingers.
 <Page18>
-"I fought with my father for three years, but in the end he said,
+"My battle with my father lasted for about three years, but in the end he said,
 'I would rather do the work of two people than train you!'
-Then he threw me out of the kitchen.
+Then he threw me out of the kitchen. // would like to remove one of the thens if possible
```

Comment:
Than and then, no conflict imo

Reply:
I was referring to this and the line below